.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,27 @@
+name: 🐛 Bug Report
+description: |
+  Describe your problem here.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: description
+    attributes:
+      label: Bug Description
+      description: Tell us what happens and what should happen instead.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of Mixxx are you running?

```

IMO a placeholder for the version would also make sense to suggest that we'd like to know the entire `major.minor.patch` version triple, not just `major.minor`.

```suggestion
      description: What version of Mixxx are you running?
      placeholder: 2.3.4, 2.4.0, ...
```